src/openrct2/ride/CableLift.cpp

```diff
@@ -356,10 +338,7 @@ static bool sub_6DF21B_loop(Vehicle* vehicle)
 
         moveInfo = vehicle_get_move_info(vehicle->TrackSubposition, vehicle->track_type, trackProgress);
         CoordsXYZ unk = { moveInfo->x, moveInfo->y, moveInfo->z };
```

Comment:
You can have the sum in the same line:
```suggestion
        auto unk = CoordsXYZ{ moveInfo->x, moveInfo->y, moveInfo->z } + vehicle->TrackLocation;
```